corehq/ex-submodules/casexml/apps/phone/fixtures.py

```diff
@@ -52,7 +52,7 @@ def __init__(self):
         if hasattr(settings, "FIXTURE_GENERATORS"):
             for group, func_paths in settings.FIXTURE_GENERATORS.items():
                 self._generator_providers[group] = filter(None, [
-                    to_function(func_path) for func_path in func_paths
+                    to_function(func_path, failhard=True) for func_path in func_paths
```

Comment:
Because `failhard` was set to False, `to_function('corehq.apps.app_manager.fixtures.report_fixture_generator')` was returning None and so this was missing from the restore. Same issue happens for `"custom.bihar.reports.indicators.fixtures.generator"`